pkg/k8s/service_cache.go

```diff
@@ -133,15 +134,15 @@ func (s *ServiceCache) GetServiceIP(svcID ServiceID) *loadbalancer.L3n4Addr {
 }
 
 // GetServiceFrontendIP returns the frontend IP (aka clusterIP) for the given service with type.
-func (s *ServiceCache) GetServiceFrontendIP(svcID ServiceID, svcType loadbalancer.SVCType) net.IP {
+func (s *ServiceCache) GetServiceFrontendIP(svcID ServiceID, svcType loadbalancer.SVCType) (net.IP, error) {
```

Comment:
Stepping back a bit, is it necessary to change the signature of this func? On an error condition, we are returning nil for the IP, so couldn't that be checked against inside `getAndUpsertPolicySvcConfig()`, instead of checking against the newly returned `error?

Reply:
Yup, that makes sense. Thanks! PTAL!